feature.env

```diff
@@ -1,9 +1,10 @@
 PORT=3007
-HMPPS_AUTH_URL=http://localhost:9091/auth
-TOKEN_VERIFICATION_API_URL=http://localhost:9091/verification
+HMPPS_AUTH_URL=http://127.0.0.1:9091/auth
```

Comment:
Out of interest is this just to work around some mac oddness? I'm sure I've had to do stuff like this in the past.

Reply:
Yes, it is. Something to do with the way the docker external ports are exposed.
I get "connection refused" from the app on localhost (but I can hit them from the browser), but it works on 127.0.0.1.